### PR TITLE
Remove references to instantx and darksend in sendcoinsdialog.cpp

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: src/qt/sendcoinsdialog.cpp [hunk 1]
```diff
@@ -70,13 +70,13 @@ SendCoinsDialog::SendCoinsDialog(const PlatformStyle *_platformStyle, QWidget *p
 
     // Dash specific
     QSettings settings;
-    if (!settings.contains("bUseDarkSend"))
-        settings.setValue("bUseDarkSend", false);
-    if (!settings.contains("bUseInstantX"))
-        settings.setValue("bUseInstantX", false);
+    if (!settings.contains("bUsePrivateSend"))
```
Comment (UdjinM6) [inline src/qt/sendcoinsdialog.cpp]:
```suggestion
    if (settings.contains("bUseDarkSend"))
        settings.setValue("bUsePrivateSend", settings.value("bUseDarkSend").toBool());
    if (!settings.contains("bUsePrivateSend"))
```

File: src/qt/sendcoinsdialog.cpp [hunk 2]
```diff
@@ -70,13 +70,13 @@ SendCoinsDialog::SendCoinsDialog(const PlatformStyle *_platformStyle, QWidget *p
...
+        settings.setValue("bUsePrivateSend", false);
+    if (!settings.contains("bUseInstantSend"))
```
Comment (UdjinM6) [inline src/qt/sendcoinsdialog.cpp]:
```suggestion
    if (settings.contains("bUseInstantX"))
        settings.setValue("bUseInstantSend", settings.value("bUseInstantX").toBool());
    if (!settings.contains("bUseInstantSend"))
```

**Comment by PastaPastaPasta** [comment]
> 
> 
> Can't just replace one with another, should check old settings first and, if they exist, re-apply their values to new ones. Otherwise it would be equivalent to partial settings reset.

I just figured resetting those settings wouldn't be that big of a deal. Afterall, you should review them every time you make a tx

File: src/qt/sendcoinsdialog.cpp [hunk 3]
```diff
@@ -70,13 +70,20 @@ SendCoinsDialog::SendCoinsDialog(const PlatformStyle *_platformStyle, QWidget *p
...
-    if (!settings.contains("bUseDarkSend"))
-        settings.setValue("bUseDarkSend", false);
-    if (!settings.contains("bUseInstantX"))
-        settings.setValue("bUseInstantX", false);
-
-    bool fUsePrivateSend = settings.value("bUseDarkSend").toBool();
-    bool fUseInstantSend = settings.value("bUseInstantX").toBool();
+    //TODO remove Darksend sometime after 0.14.1
+    if (settings.contains("bUseDarkSend"))
+        settings.setValue("bUsePrivateSend", settings.value("bUseDarkSend").toBool());
```
Comment (UdjinM6) [inline src/qt/sendcoinsdialog.cpp]: should add `settings.remove("bUseDarkSend");` (and wrap `if` contents with `{}`)

File: src/qt/sendcoinsdialog.cpp [hunk 4]
```diff
@@ -70,13 +70,20 @@ SendCoinsDialog::SendCoinsDialog(const PlatformStyle *_platformStyle, QWidget *p
...
+    if (!settings.contains("bUsePrivateSend"))
+        settings.setValue("bUsePrivateSend", false);
+
+    //TODO remove InstantX sometime after 0.14.1
+    if (settings.contains("bUseInstantX"))
+        settings.setValue("bUseInstantSend", settings.value("bUseInstantX").toBool());
```
Comment (UdjinM6) [inline src/qt/sendcoinsdialog.cpp]: same for `bUseInstantX`

**Pull request merged**